ci-operator/step-registry/gather/must-gather/gather-must-gather-ref.yaml

```diff
@@ -12,6 +12,6 @@ ref:
     requests:
       cpu: 300m
       memory: 300Mi
-  timeout: 35m30s
+  timeout: 90m
```

Comment:
90m is sufficient bc I observed a 252 worker m-g completing in 65m

Reply:
There's another timeout just above this, controlled by an env var - are you configuring that in your jobs?